README.md

```diff
@@ -1,2 +1,52 @@
-# aws-cloudwatch-firehose
-Forwards logs from cloudwatch to NewRelic through firehose
+# aws-unified-firehose
+Forwards logs from cloudwatch to NewRelic through AWS firehose
+
+## Features
+
+- Collects logs from AWS CloudWatch.
+- Forwards logs to NewRelic using AWS Firehose.
+- Allows users to attach custom attributes to the logs to make it easier to search, filter, analyze, and parse the logs
+- Scalable and reliable log forwarding.
+
```

Comment:
you can add bufferhints param detail also we are setting .

Reply:
Stores license key in Secret Manager by default.

Reply:
Added Stores license key in Secret Manager by default, not sure of bufferhints, may be we can add once it is available ?